tools/git/hooks/install_hooks.ps1

```diff
@@ -1,2 +1,3 @@
-New-Item -ItemType HardLink -Name ../../../.git/hooks/pre-commit -Value src/pre_commit.py
-New-Item -ItemType HardLink -Name ../../../.git/hooks/commit-msg -Value src/commit_msg.py
+Remove-Item -Recurse -Force ..\..\..\.git\hooks\*
```

Comment:
Would we want to consider only deleting the 2 files instead of the whole folder?  Is there any chance someone will have other stuff hanging out in there that they might want to keep?